control/init.sh

```diff
@@ -2,6 +2,7 @@
 CCHQ_VIRTUALENV=${CCHQ_VIRTUALENV:-cchq}
 VENV=~/.virtualenvs/$CCHQ_VIRTUALENV
 NO_INPUT=0
+USE_SYSTEM_PYTHON=${USE_SYSTEM_PYTHON:-false}
```

Comment:
The meaning of this will be wrong on Ubuntu 22.04, right? Maybe it should be renamed to `BIONIC_USE_SYSTEM_PYTHON`?

Reply:
See d7b796ffecf8df3ef2b5635456424a572a1c79fe